js/src/ui/Certifications/certifications.js

```diff
@@ -73,15 +64,12 @@ function mapStateToProps (_, initProps) {
 
   return (state) => {
     const certifications = state.certifications[account] || [];
-    return { certifications };
+    const dappsUrl = state.api.dappsUrl;
+    return { certifications, dappsUrl };
   };
 }
 
-function mapDispatchToProps (dispatch) {
-  return bindActionCreators({ fetchCertifications }, dispatch);
-}
-
 export default connect(
   mapStateToProps,
-  mapDispatchToProps
+  null
```

Comment:
I actually like `null` here because `null !== undefined` and because `connect` is commonly known as the function that accepts two params, one function for each direction.